src/transformStateWithClones.js

```diff
@@ -1,13 +1,58 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} newState
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newState = createClone(state);
```

Comment:
There are the easiest ways to create a copy, such as `{ ...state }` or `Object.assign({}, state)`